actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -26,7 +26,42 @@ module TagHelper
       PRE_CONTENT_STRINGS[:textarea]  = "\n"
       PRE_CONTENT_STRINGS["textarea"] = "\n"
 
+      class TagBuilder #:nodoc:
+        include TagHelper
```

Comment:
Oh, just released that I was not clear, sorry. Changing `tag` and `content_tag` to build an instance is fine as a transition and it is better in my opinion to move all methods to inside TagBuilder